ejb3/src/main/java/org/jboss/as/ejb3/remote/AssociationImpl.java

```diff
@@ -393,7 +434,7 @@ public void removedEntries(final Map<String, List<ClientMapping>> removed) {
         }
     }
 
-    static Object invokeMethod(final ComponentView componentView, final Method method, final InvocationRequest incomingInvocation, final InvocationRequest.Resolved content, final CancellationFlag cancellationFlag) throws Exception {
+    static ResultAndContextData invokeMethod(final ComponentView componentView, final Method method, final InvocationRequest incomingInvocation, final InvocationRequest.Resolved content, final CancellationFlag cancellationFlag) throws Exception {
```

Comment:
I really don't like this combined object.  It would be better to pass in the context data map as a parameter instead.